src/App.jsx

```diff
@@ -9,36 +10,32 @@ export const tabs = [
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">Selected tab is Tab 1</h1>
+export const App = () => {
+  const [activeTabId, setActiveTabId] = useState(tabs[0]?.id || '');
 
-    <div data-cy="TabsComponent">
-      <div className="tabs is-boxed">
-        <ul>
-          <li className="is-active" data-cy="Tab">
-            <a href="#tab-1" data-cy="TabLink">
-              Tab 1
-            </a>
-          </li>
+  const handleTabSelected = tabId => {
+    const selectedTab = tabs.find(tab => tab.id === tabId);
 
-          <li data-cy="Tab">
-            <a href="#tab-2" data-cy="TabLink">
-              Tab 2
-            </a>
-          </li>
+    if (selectedTab) {
+      setActiveTabId(tabId);
+    } else {
+      setActiveTabId(tabs[0]?.id || '');
+    }
+  };
 
-          <li data-cy="Tab">
-            <a href="#tab-3" data-cy="TabLink">
-              Tab 3
-            </a>
-          </li>
-        </ul>
-      </div>
+  const getActiveTabIndex = () => {
```

Comment:
i recommend to work not with the tab index here, but with the Tab title that is already provided on the `tabs` constant.

Reply:
nesta linha segui o exemplo do exercício que retorna apenas o numero da aba selecionada, fiz a mudança mas me parece errado, valida por favor se devo mandar o update conforme a imagem em anexo que já farei isto

![image](https://github.com/user-attachments/assets/45ddbcae-5812-4e15-811b-b0ced6b1b8c9)

Reply:
Imagine que o que está sendo exibido não seja o índice da página, mas sim o título dela.

![image](https://github.com/user-attachments/assets/219fd686-4ac4-4889-9758-9e82f9dcbfc6)

Caso possuíssemos uma tab com nome de `Produtos`, o texto irá exibir 

`Selected tab is Products`, faz sentido?

Reply:
faz sim ! somente a pagina de exemplo me dizia outra coisa, mas assim faz mais sentido, ja atualizo o pull request um momento, obrigado por esclarecer!

Reply:
como meu expediente hoje já foi finalizado, o próximo mentor irá revisar pra você =)

parabéns pelo código e bons estudos 🙌